catalyst-gateway/bin/src/rbac/indexing_context.rs

```diff
@@ -0,0 +1,125 @@
+//! A RBAC context used during indexing.
```

Comment:
what do you thing to move this code to be a part of the `db/index/block` mod ? 
Because it seems it only related to that logic

Reply:
The context is really used in the rbac module and it is only created on the indexing side. I prefer to live all the logic close together.